README.md

```diff
@@ -25,12 +25,14 @@ The application:
  * NUCLEO_F429ZI + ATMEL AT233 15.4 shield (mesh `LOWPAN_ROUTER` mode)
  * K64F (Ethernet)
  * NUCLEO_F429ZI (Ethernet)
- * UBLOX_EVK_ODIN_W2 (WiFi - ethernet does not seem to work)
+ * UBLOX_EVK_ODIN_W2 (WiFi & Ethernet - use the supplied configs/eth_v4.json for ethernet)
```

Comment:
Ethernet to be capitalized in the second instance, as well. You might also want to use the tics around configs/eth_v4.json to make it stand out.